### PR TITLE
Fix for https://github.com/HerculesWS/Hercules/issues/2349.

### PR DESCRIPTION
**Pull request by Kenpachi2k13**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

ipban_log() only inserts the first 3 octets of an IP and an asterisk for last one, so we can set the columns length of `list` in `ipbanlist` table to 13.

**Issues addressed:** #2349

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

File: sql-files/upgrades/2019-11-22--23-58.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+ALTER TABLE `ipbanlist` MODIFY `list` VARCHAR(13) NOT NULL DEFAULT '';
+INSERT INTO `sql_updates` (`timestamp`) VALUES (1574463539);
```
Comment (dastgirp) [inline sql-files/upgrades/2019-11-22--23-58.sql]: add this in main.sql too

Comment (Kenpachi2k13) [reply sql-files/upgrades/2019-11-22--23-58.sql]: My mistake, sorry. Done!

**Comment by 4144** [comment]
in this pr also useless merge commit

**Comment by pazkero** [comment]
Question, some custom plugins which I've saw on the forums add a command called "@ipban", for example, which is a non-automatic way of adding IP Bans.

An IP official length is 15 or 16 (xxx.xxx.xxx.xxx). Wouldn't this cause issues when IP bans are not done automatically? (Also, in some rather specific cases, it might be a desired behavior to ban only the specific IP. Yes, I am aware of how dynamic IP works and why we usually use a star.)

TL;DR Doesn't this causes issues or SQL errors if there are IP Bans of length 15?

**Comment by dastgirp** [comment]
@pazkero 
If you see the source, we always add only 3 octet and wildcard

**ipban.c:**
```c
if (SQL_ERROR == SQL->Query(ipban->sql_handle, "INSERT INTO `%s`(`list`,`btime`,`rtime`,`reason`) VALUES ('%u.%u.%u.*', NOW() , NOW() +  INTERVAL %u MINUTE ,'Password error ban')",
			ipban->dbs->table, p[3], p[2], p[1], login->config->dynamic_pass_failure_ban_duration))
```

As you can see here, the value inserted is `'%u.%u.%u.*'`.

I don't know why this is done. Also for custom source, I guess you would need to manually change sql for now. I will open the issue on your behalf as you have mentioned it.
